### PR TITLE
[GORA-655] RethinkDB datastore Implementation 

### PR DESCRIPTION
**Pull request by rumesh-perera**
This represents final PR for RethinkDB Datastore Implementation for Apache Gora. This work is done as my GSOC 2020 project for Apache.

### CODE REVIEW

File: gora-rethinkdb/src/main/java/org/apache/gora/rethinkdb/store/RethinkDBStore.java [hunk 1]
```diff
@@ -0,0 +1,774 @@
...
+        break;
+      case INT:
+        result = Integer.valueOf(obj.get(docf).toString());
+        break;
+      case LONG:
+        result = Long.valueOf(obj.get(docf).toString());
+        break;
+      case STRING:
+        result = new Utf8(obj.get(docf).toString());
+        ;
```
Comment (madhawa-gunasekara) [inline gora-rethinkdb/src/main/java/org/apache/gora/rethinkdb/store/RethinkDBStore.java:619]: remove this semi colon

**Review by madhawa-gunasekara — CHANGES_REQUESTED** [review]

File: gora-rethinkdb/src/main/java/org/apache/gora/rethinkdb/store/RethinkDBStore.java (hunk 1 above)
Comment (rumesh-perera) [reply gora-rethinkdb/src/main/java/org/apache/gora/rethinkdb/store/RethinkDBStore.java:619]: @madhawa-gunasekara Thank you. I will address this.

**Comment by djkevincr** [comment]
@rumesh-perera Thank you for your hard work during the GSoC period. PR looks very good. I m going to keep PR for a while for let others to review. Good Luck with your final evaluation.

**Comment by madhawa-gunasekara** [comment]
@rumesh-perera Thanks for the Contribution. merged your pull request.

**Pull request merged**